pkg/model/model.go

```diff
@@ -8,6 +8,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	// StrategyExisting holds the name of the existing strategy
+	StrategyExisting = "existing"
```

Comment:
What about the other strategies? Do we want to add them too?

Reply:
They are supposed to be dynamic. The only thing that would reference them would be test code. The thinking is to read in 'most' strategies from a ConfigMap to configure the Template engine. Existing is a little bit of a hack as it affects other parts of the code than the other strategies. I couldn't really find a nice way to make them all the same unless we want to introduce a second handler (but then there is a Deployment vs DeploymentConfig issue as well) of some sort like telepresence+clone.